maxim/apis/maxim_apis.py

```
@@ -170,13 +316,23 @@ def get_prompt(self, id: str) -> VersionAndRulesWithPromptId:
+        except httpx.HTTPStatusError as e:
+            if e.response is not None:
+                try:
+                    error_data = e.response.json()
+                    if (
+                        error_data
+                        and isinstance(error_data, dict)
+                        and "error" in error_data
+                        and isinstance(error_data["error"], dict)
+                        and "message" in error_data["error"]
+                    ):
+                        raise Exception(error_data["error"]["message"]) from e
```

Pylint (3.3.7)

[warning] Raising too general exception: Exception

(W0719)